### PR TITLE
Update hardcoded headers for Foundation, Ropsten, Kovan and Classic

### PR DESCRIPTION
**Pull request by Tbaut**
Update light client hardcoded headers for the main networks

- update foundation to #7262209
- update kovan to #10434561
- update ropsten to #5101569
- update classic to #7555073

edit: correct ropsten block

### CODE REVIEW

**Comment by Tbaut** [comment]
Unlike the change-view from Github may let you think, original hardcoded headers haven't been removed in `classic.json`. I simply replaced 2 spaces with a tab.

**Comment by HCastano** [comment]
I haven't been able to sync on Ropsten, but the others look good. Will approve when I get the Ropsten thing figured out.

File: ethcore/res/ethereum/ropsten.json [hunk 1]
```diff
@@ -62,8 +62,8 @@
...
-		"totalDifficulty": "17277173430188370",
+		"header": "f90213a002856c991cee4843dd0055f843dba5304a2de90db9ef16f6e06970984a147fffa01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d4934794d9ee4d08a86b430544254ff95e32aa6fcc1d3163a08631a00947e31a46b0e6eadafa5a73ff6f29e9c7088f5e440206abe7687116f6a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421b9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000832637e5834cb801837a120080845c6ab20199d883010815846765746888676f312e31302e34856c696e7578a04b5cc1554f1c30864b83b6a5bf6cf12e3b7a6c1d1150814fa0f2ac1758cffd73883701bbd33b5769fe",
+		"totalDifficulty": "18208711110298778",
```
Comment (soc1c) [inline ethcore/res/ethereum/ropsten.json]: That's the wrong chain @Tbaut 

```
2019-02-27 12:33:43  Starting Parity-Ethereum/v2.2.11-stable-8e31051c43-20190220/x86_64-linux-gnu/rustc1.32.0
2019-02-27 12:33:43  Keys path /home/user/.local/share/io.parity.ethereum/keys/ropsten
2019-02-27 12:33:43  DB path /home/user/.local/share/io.parity.ethereum/chains_light/ropsten/db/ae90623718e47d66
2019-02-27 12:33:43  Running in experimental Light Client mode.
2019-02-27 12:33:43  Inserting hardcoded block #5027841 in chain
2019-02-27 12:33:43  Debug API is not available in light client mode.
2019-02-27 12:33:43  Debug API is not available in light client mode.
2019-02-27 12:33:43  Public node URL: enode://e2ea472bcbf28c3a5784e34f6ca05cca9c87779885af013eaf3ecc4d0f213e19a20f4170b83945631507b84187368df51dcb15011da4f2c125e9a1e58f258934@192.168.0.196:30303
2019-02-27 12:33:48  Syncing #5027841 0x9a38…9efb     0.0 hdr/s      0+    0 Qed   2/50 peers   560 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
2019-02-27 12:33:53  Syncing #5027841 0x9a38…9efb     0.0 hdr/s      0+    0 Qed   1/50 peers   560 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
2019-02-27 12:33:58  Syncing #5027841 0x9a38…9efb     0.0 hdr/s      0+    0 Qed   1/50 peers   560 bytes cache 0 bytes queue  RPC:  0 conn,    0 req/s,    0 µs
```

But according to etherscan it's `0xecff91fce98c94ebed5490b1b27a45c84f5a29fbd296951c7b492b391a43b3db`

https://ropsten.etherscan.io/block/5027841

Comment (HCastano) [reply ethcore/res/ethereum/ropsten.json]: I've got a relatively up to date Ropsten node, I can update this.

Comment (Tbaut) [reply ethcore/res/ethereum/ropsten.json]: Nice one, I forgot to check that one sorry about that.

**Pull request merged**